style.css

```
@@ -1,20 +1,34 @@
 html, body {
     width: 50%;
     margin: 0 auto;
-    font-family: 'Times New Roman', Times, serif;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
 
 .content{
-    width: 300px;
+    padding: 40px;
     vertical-align:  middle;
+    background-color:#ebdbbc;
+    border: 1px black solid;
+    margin-top: 10px;
     
```

Also, let's work on CSS formatting. :) 

```suggestion
    padding: 40px;
    vertical-align: middle;
    background-color: #ebdbbc;
    border: 1px black solid;
    margin-top: 10px;
    
```